frontend/src/modules/app/ToolBar.tsx

```diff
@@ -1,25 +1,29 @@
-import { AppBar, Button, Grid } from '@material-ui/core';
+import { AppBar, Button, Grid, Typography } from '@material-ui/core';
 import { makeStyles, Theme } from '@material-ui/core/styles';
 import React, { useCallback, useEffect, useState } from 'react';
 import { shallowEqual, useDispatch, useSelector } from 'react-redux';
-import { Link, useHistory, useLocation } from 'react-router-dom';
+import { useHistory, useLocation } from 'react-router-dom';
 import { getClockTime } from '../../store/app/selectors';
-import { commitRequest } from '../../store/controller/actions';
+import { commitRequest, commitStandbyRequest } from '../../store/controller/actions';
```

Comment:
Let's rename `commitStandbyRequest` to `commitDraftRequest`

Reply:
done.